Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -138,12 +138,37 @@ YUI.add('ez-platformuiapp', function (Y) {
                     oldService.setNextViewServiceParameters(newService);
                 }
             });
-
+            if (this.get('routeConfig')) {
```

Comment:
I would put that in a dedicated method with a good name so that you directly know what it's about